content/en/docs/release-notes/release-notes-1.4.md

```diff
@@ -21,4 +21,35 @@ notAfter - rb`  where  `if cert duration < renewBefore; then rb = cert duration
 / 3; else rb = renewBefore`. This change fixes a bug where, if a certificate's
 duration is very slightly larger than `renewBefore` period, then a cert gets
 renewed at `notAfter - renewBefore` which can lead to a continuous renewal loop,
-see [`cert-manager#3897`](https://github.com/jetstack/cert-manager/issues/3897).
\ No newline at end of file
+see [`cert-manager#3897`](https://github.com/jetstack/cert-manager/issues/3897).
+
+## Updating cert-manager CRDs and stored versions of cert-manager custom resources
+
+We have deprecated `cert-manager.io/v1alpha2`, `cert-manager.io/v1alpha3`, `cert-manager.io/v1beta1`, `acme.cert-manager.io/v1alpha2`, `acme.cert-manager.io/v1alpha3`, `acme.cert-manager.io/v1beta1` APIs.
+These APIs will be removed in `cert-manager` `v1.6`.
+If you have a `cert-manager` installation that is using or has previously used these APIs you may need to update `cert-manager` custom resources and CRDs. This needs to be done before upgrading to `cert-manager` `v1.6`. (See Kubernetes docs for more detailed explanation [here](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#upgrade-existing-objects-to-a-new-stored-version))
```

Comment:
suggestion: minor wording change
nitpick: we refer to "`cert-manager`" here, but we frequently use "cert-manager" in the documentation, e.g. in https://cert-manager.io/docs/release-notes/release-notes-1.3/ - should we follow the existing style? (this applies elsewhere in the commit)

```suggestion
If you have a cert-manager installation that is using or has previously used these deprecated APIs you may need to update your cert-manager resources and CRDs. This needs to be done before upgrading to cert-manager `v1.6`. (See the [Kubernetes docs](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#upgrade-existing-objects-to-a-new-stored-version) for a more detailed explanation)
```